test/models/dish_menu_test.rb

```diff
@@ -18,15 +18,15 @@ class DishMenuTest < ActiveSupport::TestCase
     test 'get menu' do
         Dish.add_dish(name: "french toast", star: 3, mesh_soft:2, puree:"", portion_size:"", diet:"1/2 serving")
         DishMenu.add_dish_to_cycle(48,"french toast")
-        menu=DishMenu.get_menu(48)
+        menu=DishMenu.get_by_day(48)
         assert menu.length()==1
     end
 
     test 'update dish menu' do
         Dish.add_dish(name: "french toast", star: 3, mesh_soft:2, puree:"", portion_size:"", diet:"1/2 serving")
         DishMenu.add_dish_to_cycle(0,"french toast")
```

Comment:
Not sure what the testing convention is, but thoughts on abstracting out the name of the dish (ie dish_name = "french toast") in the test? Maybe there's no value in it though.

Reply:
almost certainly best practice, but I didn't write this test....